addons/web_editor/static/src/scss/wysiwyg.scss

```diff
@@ -642,13 +642,17 @@ img.o_we_selected_image {
 }
 
 .o_edit_menu_popover {
+    box-shadow: rgba(0, 0, 0 , 0.1) 0px 4px 12px, rgba(0, 0, 0, 0.05) 0px 0px 4px;
+    border-color: rgba(0,0,0,0.025);
```

Comment:
`box-shadow` and  `border-color` should be placed after position and sizing rules
ref: https://www.odoo.com/documentation/16.0/contributing/development/coding_guidelines.html#properties-order